python/extractor/tsg-python/BUILD.bazel

```diff
@@ -5,7 +5,7 @@ codeql_rust_binary(
     name = "tsg-python",
     srcs = ["src/main.rs"],
     aliases = aliases(),
-    data = ["python.tsg"],
+    compile_data = ["python.tsg"],
```

Comment:
this wasn't actually meant to work earlier, and now the rule is stricter about it.